assets/css/index.css

```diff
@@ -388,3 +388,30 @@ button.clipboard:focus {
 .script-copy {
   cursor: pointer;
 }
+
+.banner-link:hover,
+.banner-link:active {
+  color: white;
+  text-decoration: none;
+  background-color: white;
+  color: #333;
+  transition: all 0.3s;
+}
+
+.banner-link {
+  border: 1px solid white;
+  padding: 15px;
+  font-size: 24px;
+  margin-right: 20px;
+  color: white;
+}
+
+.down-link {
+  background-color: white;
+  color: #333;
+}
+
+.down-link:hover,
+.down-link:active {
+  filter: brightness(90%);
```

Comment:
Instead of changing the brightness, I think it's better if on hover the button becomes transparent and the color changes to white.

Reply:
@shahednasser I did the required changes.